### PR TITLE
[thethingsnetwork] Update the to v3

### PR DESCRIPTION
**Pull request by angelnu**
## Proposed change

Update documentation to match the updated thethingsnetwork integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113375
- Link to parent pull request in the Brands repository:  NA
- This PR fixes or closes issue: NA

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/thethingsnetwork.markdown [hunk 1]
```diff
@@ -39,106 +35,24 @@ Visit the [The Things Network Console](https://console.thethingsnetwork.org/) we
...
-  description: The access key.
-  required: true
-  type: string
-{% endconfiguration %}
-
-## Sensor
+![Application overview](/images/integrations/thethingsnetwork/applications.png)
 
-The `thethingsnetwork` sensor platform allows you to get data from a [The Things Network Storage Integration](https://www.thethingsnetwork.org/docs/applications/storage/).
+Enable the storage integration under the integrations menu:
```
Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
1. Under the integrations menu, enable the storage integration:
```

File: source/_integrations/thethingsnetwork.markdown [hunk 2]
```diff
@@ -39,106 +35,24 @@ Visit the [The Things Network Console](https://console.thethingsnetwork.org/) we
...
 
-This platform requires that the [The Things Network integration](#configuration) is set up and the [The Things Network Storage Integration](https://www.thethingsnetwork.org/docs/applications/storage/) as well.
+![Storage Integration](/images/integrations/thethingsnetwork/storage_integration.png)
 
-### Prerequisites
+Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.
```
Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
2. Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.
```

File: source/_integrations/thethingsnetwork.markdown [hunk 3]
```diff
@@ -39,106 +35,24 @@ Visit the [The Things Network Console](https://console.thethingsnetwork.org/) we
...
 
-Visit the [The Things Network Console](https://console.thethingsnetwork.org/) website, log in with your The Things Network credentials, choose your application from **Applications** and go to **Integrations**.
+![Payload Formatters](/images/integrations/thethingsnetwork/payload_formatters.png)
 
-Add a new integration.
+You need an API key to be able to read the data from your application. The minimum rights is `Read Application Traffic (uplink and downlink)`.
```
Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
1. You need an API key to be able to read the data from your application. 
  - The minimum rights is `Read Application Traffic (uplink and downlink)`.
```

File: source/_integrations/thethingsnetwork.markdown [hunk 4]
```diff
@@ -39,106 +35,25 @@ Visit the [The Things Network Console](https://console.thethingsnetwork.org/) we
...
+![Storage Integration](/images/integrations/thethingsnetwork/storage_integration.png)
 
-### Prerequisites
+2. Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.
 
-Visit the [The Things Network Console](https://console.thethingsnetwork.org/) website, log in with your The Things Network credentials, choose your application from **Applications** and go to **Integrations**.
+![Payload Formatters](/images/integrations/thethingsnetwork/payload_formatters.png)
 
-Add a new integration.
+1. You need an API key to be able to read the data from your application.
```
Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
3. You need an API key to be able to read the data from your application. 
```

File: source/_integrations/thethingsnetwork.markdown [hunk 5]
```diff
@@ -39,106 +35,25 @@ Visit the [The Things Network Console](https://console.thethingsnetwork.org/) we
...
 
-### Prerequisites
+2. Ensure you have an [Uplink Payload Formatter](https://www.thethingsindustries.com/docs/integrations/payload-formatters/) for your device.
 
-Visit the [The Things Network Console](https://console.thethingsnetwork.org/) website, log in with your The Things Network credentials, choose your application from **Applications** and go to **Integrations**.
+![Payload Formatters](/images/integrations/thethingsnetwork/payload_formatters.png)
 
-Add a new integration.
+1. You need an API key to be able to read the data from your application. 
+  - The minimum rights is `Read Application Traffic (uplink and downlink)`.
```
Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
  - The minimal required right is `Read Application Traffic (uplink and downlink)`.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @angelnu :+1:

**Comment by joostlek** [comment]
Huh, how did this get merged when I just merged the core PR? cc @c0ffeeca7

**Comment by joostlek** [comment]
Aaah https://github.com/home-assistant/core/pull/113375#issuecomment-2132246905

It was reverted

**Pull request merged**